kover-maven-plugin/docs/index.md

```diff
@@ -0,0 +1,333 @@
+
+        <!-- Property name to pass java agent argument to the JVM in which tests are running. -->
+        <!-- String, default: argLine -->
+        <agentPropertyName>argLine</agentPropertyName>
+        
+        <!-- Filters to limit the code that gets into the report. -->
+        <!-- Used in goals: report-xml, report-html, verify, report-ic, log -->
+        <!-- Complex type, default: no filters -->
+        <filters>
+            <filters>
+                <!-- Specify classes and methods that should be excluded from reports -->
+                <!-- Excludes have priority over includes. -->
+                <excludes>
+                    <!-- Exclude specified classes from report -->
+                    <!-- Using fully-qualified JVM class names, wildcards `*` and `?` are acceptable. -->
+                    <!-- List of String values, default: empty list -->
+                    <classes>com.example.*.ExcludedByName,com.example.*.serializables.*$Companion</classes>
+
+                    <!-- Exclude classes marked by specified annotations -->
+                    <!-- Using fully-qualified JVM class names, wildcards `*` and `?` are acceptable. -->
+                    <!-- List of String values, default: empty list -->
+                    <annotatedBy>*.Generated</annotatedBy>
+
+                    <!-- Exclude classes inherited from the specified classes, or implementing the specified interfaces. -->
+                    <!-- Using fully-qualified JVM class names, wildcards `*` and `?` are acceptable. -->
+                    <!-- List of String values, default: empty list -->  
+                    <inheritedFrom>java.lang.AutoCloseable</inheritedFrom>
+
+                    <!-- Exclude classes of specified projects by project names. -->
+                    <!-- Wildcards `*` and `?` are acceptable. -->
+                    <!-- List of String values, default: empty list -->
+                    <projects>test-utils</projects>
+                </excludes>
+                
+                <!-- Specify classes that should be included to reports, all other classes are not included in the report. -->
+                <!-- Excludes have priority over includes. -->
+                <includes>
+                    <!-- content is the same as in excludes block -->
+                </includes>
+            </filters>
+        </filters>
+
+        <!-- Specify to collect coverage from project dependencies. -->
+        <!-- Used in goals: report-xml, report-html, verify, report-ic, log -->
+        <!-- Boolean, default: false -->
+        <aggregate>true</aggregate>
+        
+        <!-- Binary reports that built in advance, before the start of the project build. -->
+        <!-- Used in goals: report-xml, report-html, verify, report-ic, log -->
+        <!-- List of File values, default: empty list -->
+        <additionalBinaryReports>${project.basedir}/external/report.ic</additionalBinaryReports>
```

Comment:
For distributed test runs, yes, it is convenient

Reply:
Are you planning to write a documentation for such use case?

Reply:
Not now, because users can use this opportunity in different cases, and so far there is not enough information on how to describe it applicable to everyone.

So far, this setting is a way of enabling users who will be dealing with such a problems and will create an issue with the question.